### ISSUE TITLE
Get 18F access to CMS's internal wiki

### ISSUE THREAD

**Issue by NAretakis** [CMSgov/CMCS-DSG-DSS-Oversight#14]
**Who is the user?** 
18F team; SOs 

**What will this help them do?** 
learn about existing setup of wiki, so that we can structure how our work gets delivered to state officers to be as user-friendly as possible and optimize for successful onboarding & adoption

**When is it needed?** 
Not for a month or two, but we want to start the process now!

**What steps are needed to make it a reality?** 

- [x] Send 18F exports from the wiki that help give context to how it is organized and used
- [x] Liz or Nick look into what it will take to start the access process, and let Amy & Alex know what they need to do
- [x] Send 18F exports of wiki content in the meantime
- [x] Get EUA account for 18F staff
- [ ] Add job codes for SharePoint access
- [ ] Confirm 18F access to SharePoint

**Comment by NAretakis** [comment]
EUA access is needed for Sharepoint. 18F folks need to use this to get the ball rolling: https://eua.cms.gov/efi/signup

**Comment by amyashida** [comment]
Princess and I are here in the process: ![image.png](https://images.zenhubusercontent.com/563a2ca4638e718c995e4667/72eef3f3-f0d1-4127-b008-ad737ea728be)

**Comment by NAretakis** [comment]
Ok, tracked down your accounts. I am now reaching out the federal approver to see if they can approve. Once that is done and your accounts are setup, we then have to get the right Job Codes added to your account.

**Comment by alexpandel** [comment]
working on this as well! (late to the party as usual 🙈)

**Comment by amyashida** [comment]
Looks like Princess and I are approved! Alex is in progress.

**Comment by NAretakis** [comment]
Working on figuring out the right job codes for access. I suspect our VPN will also be needed.

**Comment by alexpandel** [comment]
omg I am finally at this stage too, sorry for the HUGE delay! will update when I get the approval notice 🤞
![Screen Shot 2020-08-25 at 11 48 00 AM](https://user-images.githubusercontent.com/952401/91215153-f797f480-e6c8-11ea-88cb-b5513fec8e50.png)

**Comment by alexpandel** [comment]
reassigning this to 18F now since @NAretakis identified some next steps for us, woo! copying over from Teams: "Do this first: https://eua.cms.gov/efi

Once the application is approved by the COR in EFI the new EUA account is created and the application moves on to Security for the background investigation. Eghosa is our COR so we just need to loop in him into the EFI request." (the rest of the instructions are in our Teams space with us + Nick from today; wasn't sure how much was ok to be public on Github)

**Comment by nickarrow** [comment]
Closing this, I think we have enough workarounds in place that this is no longer needed.
